<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsmServerWidget</class>
 <widget class="QWidget" name="OsmServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbOsmServerSelected">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edOsmServerUrl"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbOsmServerAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/AllIcons.qrc">
         <normaloff>:/Icons/actions/list-add.png</normaloff>:/Icons/actions/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbOsmServerDel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/AllIcons.qrc">
         <normaloff>:/Icons/actions/list-remove.png</normaloff>:/Icons/actions/list-remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Authentication type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="authType">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Username and password (deprecated)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OAuth2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="credentials">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Page1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1,3">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edOsmServerUser"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Pwd:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edOsmServerPwd">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbLoginState">
         <property name="text">
          <string>Login state: Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>309</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="tbOAuth2Login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons/AllIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>authType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>credentials</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
